numpy/core/tests/examples/limited_api/setup.py

```diff
@@ -1,22 +0,0 @@
-"""
-Build an example package using the limited Python C API.
-"""
-
-import numpy as np
-from setuptools import setup, Extension
-import os
-
-macros = [("NPY_NO_DEPRECATED_API", 0), ("Py_LIMITED_API", "0x03060000")]
-
-limited_api = Extension(
-    "limited_api",
-    sources=[os.path.join('.', "limited_api.c")],
-    include_dirs=[np.get_include()],
-    define_macros=macros,
```

Comment:
This happens to work (perhaps, the test doesn't actually check that the extension is `.abi3`), but this seems wrong. `setuptools.Extension` expects a `py_limited_api=True` keyword: https://github.com/pypa/setuptools/blob/f7532b24c49b8285496734b544ea12a6fb6fdc52/setuptools/extension.py#L119.